test/com/amazon/ionschema/SchemaImportTest.kt

```diff
@@ -46,6 +46,14 @@ class SchemaImportTest {
         }
     }
 
+    @Test
+    fun getImport_new_schema_from_symbol() {
+        val test = "schema_header::{ imports: [ {id: 'schema/import/abcde.isl' }] } schema_footer::{}"
+        val ion = IonSystemBuilder.standard().build()
+        val iss = IonSchemaSystemBuilder.standard().addAuthority(AuthorityFilesystem("ion-schema-tests")).withIonSystem(ion).build()
+        val schema = iss.newSchema(ion.iterate(test))
```

Comment:
Let's do something with the returned value (`assertNotNull`, maybe? Or something more robust) to make sure it looks right.